cbopensource/connectors/taxii/bridge.py

```diff
@@ -41,6 +40,26 @@ def total_seconds(td):
     except:
         return 0
 
+def score_indicator(indicator, default_score=25):
+    if indicator.confidence:
+
+        if str(indicator.confidence.value).isdigit():
+            #
+            # Get the confidence score and use it for our score
+            #
+            score = int(indicator.confidence.to_dict().get("value", default_score))
+        else:
+            if str(indicator.confidence.value).lower() == "high":
```

Comment:
You're converting to a string and then performing lower() up to three times.  Seems like this could be done more efficiently.